alembic/testing/assertions.py

```diff
@@ -98,8 +101,8 @@ def expect_raises(except_cls, check_context=True):
     return _expect_raises(except_cls, check_context=check_context)
 
 
-def expect_raises_message(except_cls, msg, check_context=True):
-    return _expect_raises(except_cls, msg=msg, check_context=check_context)
+def expect_raises_message(except_cls, msg, check_context=True, text_exact=False):
```

Comment:
This function was incorrectly used for text exact checking for ValueError messages. The raw message was assumed to be a regexp but it was a simple string. So it was passed by coincidence.